### ISSUE TITLE
Update forms once authentication is complete

### ISSUE THREAD

**Issue by BCerki** [bcgov/cas-registration#191]
#### Describe the task

Because we started building forms before we set up authentication, users have to enter their operator when adding/editing operations. If they log in with a bceid, we already know their company's information, so this doesn't need to be included in the form.

Additionally, this ticket is about confirming that users can only see their companies' information.

#### Acceptance Criteria

- [ ] update operation form
- [ ] update other? forms
- [ ] change requests as needed so users only see info appropriate to their role and company

#### Additional context

- Add any other context about the task here.
- Or here

**Comment by andrea-williams** [comment]
for refinement: partially a duplicate of other tickets re: authentication-protected API endpoints, but we still want to keep this ticket because it would be a user convenience to pre-populate some Operator fields based on their Business BCeID account into

**Comment by Sepehr-Sobhani** [comment]
Just had a conversation with @hannavovk and @BCerki. We were discussing the `prime admin access request feature` and a question came up. Do we only want to display a list of operators who are associated with IO's BCeID, or do we want to show all available operators in the DB?
cc: @andrea-williams @suhafa

**Comment by andrea-williams** [comment]
When a logged-in (with Business BCeID) user wants to request prime admin access, yes they should be able to see the list of all available operators from the DB. The rationale being that there isn't an easy way to exactly match up the operator name that may be tied to the Business BCeID account with operator names in our DB. (Example: Business BCeID account has name Western Forest Products, operator name in database is Western Forest Products Ltd.)

**Comment by Dianadec** [comment]
May not be needed anymore cc @BCerki . Closing.
